test/Api.Test/Controllers/AccountsControllerTests.cs

```diff
@@ -91,14 +91,14 @@ public async Task PostPrelogin_WhenUserExists_ShouldReturnUserKdfInfo()
     }
 
     [Fact]
-    public async Task PostPrelogin_WhenUserDoesNotExist_ShouldDefaultToSha256And100000Iterations()
+    public async Task PostPrelogin_WhenUserDoesNotExist_ShouldDefaultToSha256And600000Iterations()
     {
         _userRepository.GetKdfInformationByEmailAsync(Arg.Any<string>()).Returns(Task.FromResult((UserKdfInformation)null));
 
         var response = await _sut.PostPrelogin(new PreloginRequestModel { Email = "user@example.com" });
 
         Assert.Equal(KdfType.PBKDF2_SHA256, response.Kdf);
-        Assert.Equal(100000, response.KdfIterations);
+        Assert.Equal(600000, response.KdfIterations);
```

Comment:
Instead of testing a magic number should we use `AuthConstants.PBKDF2_ITERATIONS.Default`

Reply:
I'm split. On one hand I think it's good to have a failing test to catch changes on the other hand it's more noise.

Reply:
The only reason for this test to change is when we update the iterations, so I wouldn't expect it to fail because someone bumped the DefaultIterations.
We want them to match, that what we are testing.